rigex-front-end/src/components/rigs/noRigs.js

```diff
@@ -0,0 +1,26 @@
+import AddIcon from '@material-ui/icons/Add';
+
+import './noRigs.scss';
+import rigImage from '../../assets/rig.svg';
+
+const NoRigs = () => {
+  return (
+    <div className="mainPanel">
+      <div className="imgContainer">
+        <img src={rigImage} alt="rig svg" />
+      </div>
+      <div className="txtContainer">
+        <span className="addRigsText">
+          Start organizaing your rigs & wells!
+        </span>
+        <span className="smText">Get started by creating your first rig.</span>
+        <button className="addRigButton">
```

Comment:
studio convention is to have your styles be add-rig-button instead of addRigButton

Reply:
thanks - fixed